bciers/apps/reporting/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Install dependencies only when needed
+FROM docker.io/node:lts-alpine as deps
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /usr/src/app
+COPY ./package*.json ./
+RUN yarn install --frozen-lockfile --production==false
+
+# Production image, copy all the files and run next
+FROM docker.io/node:lts-alpine as runner
+RUN apk add --no-cache dumb-init
+ENV NODE_ENV production
+ENV PORT 3000
+WORKDIR /usr/src/app
+COPY --from=deps /usr/src/app/node_modules ./node_modules
+COPY --from=deps /usr/src/app/package.json ./package.json
+COPY ./public ./public
+COPY ./.next ./.next
+RUN chown -R node:node .
+USER node
```

Comment:
There is a thing with OpenShift where it's restricting which user IDs are allowed to run things on a container...This may or may not run on the cluster, we'll have to keep an eye on it

Reply:
When I did a test deploy, the container booted up successfully and was reachable through the route. I followed the _Nx_ documentation for setting up this container to be sure that it worked, but I'm sure I could adapt it to the way we've done it already with _registration_. Thankf for flagging it!